openapi-v1.yaml

```diff
@@ -5287,6 +5320,80 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /notebooks/{namespace}/{array}/copy:
+    parameters:
+      - type: string
```

Comment:
`{array}` path param needs to be added

Reply:
added

Reply:
We need to add

```
- type: string
  name: array
  in: path
  required: true
  description: The name of the notebook
````

Trying to generate the client returns an error

```
The swagger spec at "./TileDB-Cloud-API-Spec/openapi-v1.yaml" is invalid against swagger specification 2.0. see errors :
- path param "{array}" has no parameter definition
```